docs/source/concepts/FittingMinimizersWeightedComparisonDetailed.rst

```diff
@@ -55,19 +65,19 @@ Comparison in terms of run time
 
 Summary, median ranking
 ^^^^^^^^^^^^^^^^^^^^^^^
-.. include:: minimizers_comparison/v3.7.0/comparison_weighted_v3.7_runtime_summary.txt
+.. include:: minimizers_comparison/v3.8.0/comparison_weighted_v3.8_runtime_summary.txt
 
 Run time for individual NIST problems, "lower" difficulty
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. include:: minimizers_comparison/v3.7.0/comparison_weighted_v3.7_runtime_nist_lower.txt
+.. include:: minimizers_comparison/v3.8.0/comparison_weighted_v3.8_runtime_nist_lower.txt
 
 Run time for individual NIST problems, "average" difficulty
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. include:: minimizers_comparison/v3.7.0/comparison_weighted_v3.7_runtime_nist_average.txt
+.. include:: minimizers_comparison/v3.8.0/comparison_weighted_v3.8_runtime_nist_average.txt
 
 Run time for individual NIST problems, "higher" difficulty
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-.. include:: minimizers_comparison/v3.7.0/comparison_weighted_v3.7_runtime_nist_higher.txt
+.. include:: minimizers_comparison/v3.8.0/comparison_weighted_v3.8_runtime_nist_higher.txt
```

Comment:
Are there tables for CUTEst and Neutron Data for this section? (run time comparison, weighted)